crypto/Pos/Crypto/Signing/Signing.hs

```diff
@@ -8,21 +8,20 @@ module Pos.Crypto.Signing.Signing
 
        -- * Signing and verification
        , sign
-       , checkSig
+       , checkSig                      -- reexport
        , fullSignatureHexF
        , parseFullSignature
-
        , mkSigned
 
        -- * Versions for raw bytestrings
        , signRaw
-       , verifyRaw
+       , checkSigRaw                   -- reexport
```

Comment:
So all of this was duplicated? ..
Does https://iohk.myjetbrains.com/youtrack/issue/CSL-1157 have any relation to it?

Reply:
No relation.

I'm not sure what you mean by “duplicated”.